doc/coq.rst

```diff
@@ -134,13 +137,15 @@ The semantics of the fields are:
   Interproject composition has been available since :ref:`Coq lang
   0.4<coq-lang>`.
 
-  As of today, Dune cannot depend on installed Coq theories. This restriction
-  will be lifted in the future. Note that composition with the Coq standard
-  library is supported, but in this case the ``Coq`` prefix has been made
-  available in a qualified way, since :ref:`Coq lang 0.2<coq-lang>`.
+  Composition with installed theories has been available since :ref:`Coq lang
+  0.8<coq-lang>`. Instaled theories do not have to be installed using Dune. Dune
```

Comment:
I would certainly move the second sentence to its own section, IMHO the way we lookup installed theories deserves its own entry.

Reply:
Typo *instaled